Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/contrib",
-	"GoVersion": "go1.4.2",
+	"GoVersion": "go1.5.1",
```

Comment:
we haven't updated the rest of the project to 1.5 yet, so we should probably do this with 1.4.2 to make sure it works.